src/App.tsx

```diff
@@ -28,6 +29,12 @@ const ScrollToTop = () => {
 const App = () => {
   useEffect(() => {
     displayCareConsoleArt();
+    if (
+      import.meta.env.REACT_DISABLE_PATIENT_LOGIN === "true" &&
+      window.location.pathname === "/"
```

Comment:
let's not detect current path using `window.*`. use raviger's (the routing library in-use) to detect the current path.